diy/imu-comparison.md

```diff
@@ -159,9 +160,28 @@ Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star-
 
 `Comment: Still in very early stages of testing.`
 
+---
+# Addendum
+
+## How can I check if I have an acceptable magnegic environment?
+
+You can check by downloading any magnetometer app that show what your magnetic field strength is in uT. An option available on both iOS and Android is the app, Physics Toolbox Magnetometer. If you do use Physics Toolbox Magnetometer, you only need to pay attention to the total, not the X, Y, or Z components. Most phones have a magnetometer, but if yours does not, then there is no way to be exactly sure of your magnetic environment, but you can make some educated assumptions.
+
+## My app show around X uT is that okay?
+
+There's no one value that's acceptable. What matters is that the range of values is low. There is currently limited data to give an exact range, but a good baseline seems to be a range of less than or equal to 5 uT. 
+
+## What determines a "poor magnetic environment"?
+
+Often things made of steel or other ferromagnetic materials contribute most to a poor magnetic environment. Some common examples of things that might affect your magnetic environment include, but are not limited to: spring mattresses, radiators, PC cases, desktop speakers, or furniture that's made of steel. In most cases, the effect that these things will have extend about 6-12 inches (15-30 cm) and within that range may cause the IMU to rotate incorrectly. The size and amount of mass directly impacts the size of the effected area; a paper clip might only affect your IMU if it's directly next to it, whereas a steel bedframe might affect an area 6-12 inches (15-30 cm) away as previous mentioned. In most cases, depending on the size of your playspace, these issues of certain objects causing interference can be mitigated by avoiding or reposition them. Regardless, other factors such as the wiring or rebar in your building could also affect your magnetic environment. These last few examples are harder predict and illustrate why it's important to test with an app before assuming you might have a stable magnetic environment.
+
+## Can I still use my IMU with a magnetometer if I don't have a stable magnetic environment?
+
+Yes... This cannot be recommended, however. IMUs with magnetometers such as the MPU9250 and ICM20948 can be run without the magnetometer; the MPU6500 or MPU6050 firmware can be used on the MPU9250 instead, and the ICM20948 can run in 6DOF mode. Bear in mind, in these modes without the magnetometer these IMUs perform much worse.
```

Comment:
Don't just start with Yes, people speed reading will see that and move on.  Make sure the first few words spell out an answer that has a yes no answer